common/models/event.json

```diff
@@ -42,6 +42,11 @@
       "foreignKey": "eventId",
       "through": "EventVolunteer",
       "keyThrough": "volunteerId"
+    },
+    "eventvolunteers": {
```

Comment:
Can we pull this out? If we really need it, let's at least have it in the separate changes that need it.